content/_code-samples/use-tickets/py/use-tickets-multisig.py

```diff
@@ -0,0 +1,130 @@
+from xrpl.models.transactions import TicketCreate, AccountSet, SignerListSet, SignerEntry
+from xrpl.transaction import safe_sign_and_submit_transaction, autofill, multisign, sign
+from xrpl.models.requests.account_objects import AccountObjects, AccountObjectType
+from xrpl.models.requests.submit_multisigned import SubmitMultisigned
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.clients import JsonRpcClient
+
+# This code sample lists which tickets are outstanding against an account
+# and use the ticket for a multisignature transaction.
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/tickets.html#tickets
+# https://xrpl.org/set-up-multi-signing.html#set-up-multi-signing
+# https://xrpl.org/send-a-multi-signed-transaction.html#send-a-multi-signed-transaction
+
```

Comment:
For now, can we add a comment saying this will only work with xrpl-py version [v2.0.0-beta.0](https://github.com/XRPLF/xrpl-py/releases/tag/v2.0.0-beta.0)[v2.0.0-beta.0](https://github.com/XRPLF/xrpl-py/releases/tag/v2.0.0-beta.0) or later - that way people who try it out have some way of figuring out why it's not working? (We can open a separate PR later to remove that comment)